### DISCUSSION TITLE
Help needed ingest pfsense suricata logs into SO

### DISCUSSION THREAD

**Discussion by tbaror** [Security-Onion-Solutions/securityonion#5825] [Unsupported Versions]
Hello ,

i am trying to understand what is the right process for ingesting Suricata into SO , i have made filebeat installation and i used to ingest into my own ELK , filebeat >> logstash >> es ,now  i would like to  turn the feed into SO server , but i cant figure out how to do it .
Should i feed directly into ES or logstatsh what is the right config , in the prior config i had my own logsatsh file rules , should i use it with SO?
If some could  elaborated example howto really be  appreciated  
Please advice 
Thanks you

**Comment by dougburks** [comment]
Security Onion already includes Suricata, so if it's monitoring your traffic from a tap or span port, then all of this work is already done for you.
